tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Find required providers
+  set_fact:
+    providers: |
```

Comment:
This should be `__certificate_providers` - unless you intend this to be a variable exported for use elsewhere in the playbook, then use `certificate_providers`

Reply:
63629e5b05dcf8d1f59a5dc7ca39f9551bc6dfce